tests/backend_test_base.py

```diff
@@ -162,7 +164,7 @@ def run_test_case(self, feed_dict, input_names_with_port, output_names_with_port
     @staticmethod
     def trigger(ut_class):
         parser = argparse.ArgumentParser()
-        parser.add_argument('--backend', default="onnxruntime",
+        parser.add_argument('--backend', default=Tf2OnnxBackendTestBase.BACKEND,
```

Comment:
if --backend is specified via cmd line, it will be used as backend.
otherwise, it will try to get backend from env, and fall back to the default backend onnxruntime.